keystone_spassword/contrib/spassword/mailer.py

```diff
@@ -63,16 +63,19 @@ def send_email(self, to, subject, text):
                 CONF.spassword.smtp_server, CONF.spassword.smtp_port))
```

Comment:
Suggestion for Changelog:

```
Fix check configuration for smtp_tls, smtp_user and smtp_password as STMP server connection
```

(This was the one used by PR #80 on release/1.4.0)